### ISSUE TITLE
Squeezer 1.1.1 Spotify App shows no content

### ISSUE THREAD

**Issue by kaaholst** [kaaholst/android-squeezer#106]
```
What steps will reproduce the problem?
1.Open 'My Apps'
2.Select 'Spotify' (make sure the app is installed!)
3.

What is the expected output? What do you see instead?

Should see a menu of the various Spotify Functions: Top 100, What's New, Library, Playlists,
Radio, Recent Artists, Recent Albums,Recent Searches, Search.

What is seen is an almost empty display. See attached. Also the Spotify App icon which
is displayed looks to be a generic one rather then the 'Spotify Core' one.

What version of the product are you using? On what operating system?

Squeezer 1.1.1 on Android 4.4.4 , LMS 7.8.0 on Windows 8.1, Triode Spotify Plugin 2.3.8

Please provide any additional information below.

Spotify App works normally when started from player (Squeezebox Boom, SB3) or from
LMS Web UI.

Thanks, Squeezer is great. Hope this will soon be working.

```

Original issue reported on code.google.com by `nokeeffe8` on 2014-10-23 15:04:54

<hr>
- _Attachment: Screenshot_2014-10-23-15-34-10.png<br>![Screenshot_2014-10-23-15-34-10.png](https://storage.googleapis.com/google-code-attachments/android-squeezer/issue-80/comment-0/Screenshot_2014-10-23-15-34-10.png)_
- _Attachment: Screenshot_2014-10-23-15-36-29.png<br>![Screenshot_2014-10-23-15-36-29.png](https://storage.googleapis.com/google-code-attachments/android-squeezer/issue-80/comment-0/Screenshot_2014-10-23-15-36-29.png)_

**Comment by kaaholst** [comment]
```
It also looks like the same issue when accessing the Google Music plugin, which also
resides in My Apps.
```

Original issue reported on code.google.com by `dswegen` on 2014-11-04 16:05:18

**Comment by kaaholst** [comment]
```
Spotify plugin works if you uncheck "Show in My Apps" in Spotify under "Advanced".
It then shows up under "Internet Radio" instead, with correct icon and working menus.
One issue I've found though, is the inability to play whole playlists, there's no context
menu for them,must queue every song instead, major hassle - see issue #54 I've created
here! 
```

Original issue reported on code.google.com by `jerano` on 2014-12-16 22:11:18

**Comment by kaaholst** [comment]
Fixed in 2.0.0

Original issue reported on github.com/nikclayton/android-squeezer by kaaholst on 2020-03-07T19:29:27Z
